addons/payment/views/payment_templates.xml

```diff
@@ -339,30 +339,30 @@
         </form>
     </template>
 
-    <!-- Expandable payment icon list -->
-    <template id="icon_list" name="Payment Icon List">
-        <ul class="payment_icon_list float-end list-inline" data-max-icons="3">
-            <t t-set="icon_index" t-value="0"/>
-            <t t-set="MAX_ICONS" t-value="3"/>
-            <!-- === Icons === -->
-            <!-- Only shown if in the first 3 icons -->
-            <t t-foreach="provider.payment_icon_ids.filtered(lambda r: r.image_payment_form)" t-as="icon">
-                <li t-attf-class="list-inline-item{{'' if (icon_index &lt; MAX_ICONS) else ' d-none'}}">
-                    <span t-esc="icon.image_payment_form"
+    <!-- Expandable payment method list -->
+    <template id="method_list" name="Payment Method List">
+        <ul class="payment_method_list float-end list-inline" data-max-icons="3">
```

Comment:
I feel like this is unused and redundant with the constant defined below don't you think ?
```suggestion
        <ul class="payment_method_list float-end list-inline">
```

Reply:
It's used to being able to show icons after clicking show less during payment. Without this no icons are showing.

Reply:
I see. I didn't know it was used for that ^^ In that case, let's use the same nomenclature as the constant and call it `data-max-icons-displayed`

Reply:
Okay, done :)